### DISCUSSION TITLE
Ads  and subscription

### DISCUSSION THREAD

**Discussion by gradfrank** [rrirower/DP2SceneGraphWizard#29] [Q&A]
hello
i just wh
![gvmh](https://github.com/rrirower/DP2SceneGraphWizard/assets/121139421/c53ba9a1-8110-498c-a8a3-896af94ecc85)
at to ask when we Ads  and subscription we be ready

**Comment by rrirower** [comment]
I do not have a timeframe for the implementation of Ads and Subscriptions.  I am currently working on writing the code to implement Ads.

**Comment by fronderblash** [comment]
Hello
I saw a notification that this issue has closed 

Have the ads and subscription ready to use?

**Reply by ferdiworks** [reply]
No it's still in development!
